vectortile/data/DataGen/java-vector-tile.java

```diff
@@ -0,0 +1,230 @@
+/* THIS WILL NOT RUN
+ * Below, there are a few examples of how the test files were generated using
+ * java-vector-tile.
+ * https://github.com/ElectronicChartCentre/java-vector-tile
+ */
```

Comment:
Probably should add a license header to this, with the proper copyright attribution.